docusaurus-docs/jupyterlab-conda-store/introduction.md

```diff
@@ -1,7 +1,71 @@
 ---
-sidebar_position: 1
```

Comment:
Seems a little odd to show the sidebar at all:

![image](https://github.com/conda-incubator/conda-store/assets/317883/ff8e18f8-cba8-448b-8532-32fdcd5140ed)

Reply:
Yeah, I agree, but Docusaurus currently doesn't have a way for us to do this with our setup.

First, there is no way to "hide" the page fully. We can also set the sidebar as "collapsed" by default.
BUT - that's only possible to do globally. We can't do this because we're using the [multi-instance](https://docusaurus.io/docs/docs-multi-instance) feature to have all docs in the same place, and thesidebar is useful for conda-store & conda-store-ui.

Does this makes sense?

If we feel strongly, one alternative solution is to have a custom static page instead of using Docusaurus-generated pages.

Reply:
No, don't feel strongly, just wasn't sure if it had been noticed.  :)